src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/EnumGroupAndName.cs

```diff
@@ -10,8 +14,12 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     /// </summary>
     public struct EnumGroupAndName
     {
+        private IStringLocalizer _stringLocalizer;
+        private FieldInfo _fieldInfo;
+        private string _name;
+
         /// <summary>
-        /// Initializes a new instance of the EnumGroupAndName structure.
+        /// Initializes a new instance of the EnumGroupAndName structure. Should not be used if localization is in use.
```

Comment:
What does "in use" mean? This method should not be used in any site where users may choose multiple locales.